logger/drain/factory_test.go

```diff
@@ -0,0 +1,66 @@
+package drain
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestGetUsingInvalidValues(t *testing.T) {
+	drainStrs := []string{"bogus"}
+	for _, drainStr := range drainStrs {
+		_, err := NewDrain(drainStr)
+		if err == nil ||
```

Comment:
So, personally, I'm not opposed to longer lines and would agree with you on this.  However, I decided to follow a precedent that was set by the controller's style checks (which is Python, of course) that lines not exceed 99 characters.  There are one or two other spots in the code that break after a logic operator as well.  I'm making a judgement call to follow the precedent instead of hunting down the offending occurrences here.